docs/source/introduction.rst

```diff
@@ -0,0 +1,55 @@
+=====
+About
+=====
+
+Overview
+---------
+
+Free Unified Rendering for Python
+
+
+What is FURY ?
+---------------
+
+
+
+Mission Statement
+------------------
+
+The purpose of FURY is to make it easier to do better diffusion MR imaging research. Following up with the nipy mission statement we aim to build software that is
```

Comment:
The purpose of FURY is to make it easier to perform advanced visualizations and animations. We aim to build software that is